lib/deleteFile.integration.spec.js

```diff
@@ -0,0 +1,32 @@
+const extension = require('./index.js');
+const fs = require('fs');
+const path = require('path');
+const crypto = require('crypto');
+require('dotenv').config();
+const deleteFile = require("./deleteFile.js");
+const uploadImage = require("./uploadImage.js");
+
+var deleteid = '';
+const ACCOUNT_ID = process.env.CLOUDFLARE_ACCOUNT_ID;
+const API_KEY = process.env.CLOUDFLARE_API_KEY;
+
+describe('deleteFile', () => {
```

Comment:
where is the test for deleting a video?

Reply:
added delete video test with option to expand for failure tests